docs/pages/Accessibility.js

```diff
@@ -1,7 +1,8 @@
 // @flow strict
 import type { Node } from 'react';
-import PageHeader from './components/PageHeader.js';
-import MainSection from './components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
```

Comment:
nice

Reply:
Does this mean we can now group Gestalt component files?

Reply:
@AlbertCarreras Yes, but I think we already could do that in the past. Main thing here is that Next.js uses file based routing: https://nextjs.org/docs/routing/introduction